.test-infra/jenkins/job_PreCommit_Portable_Python.groovy

```diff
@@ -24,7 +24,10 @@ PrecommitJobBuilder builder = new PrecommitJobBuilder(
     gradleTask: ':portablePythonPreCommit',
     triggerPathPatterns: [
       '^model/.*$',
-      '^runners/.*$',
+      '^runners/core-construction-java/.*$',
+      '^runners/core-java/.*$',
+      '^runners/flink/.*$',
```

Comment:
We should also add 
`'^runners/extensions-java/.*$'`
`'^runners/reference/.*$'`

runner/reference contains PortableRunner used to submit pipelines.
runners/extensions-java contains metrics

Reply:
done